verification/curator-service/ui/src/components/LinelistTable.tsx

```diff
@@ -314,55 +318,92 @@ class LinelistTable extends React.Component<Props, LinelistTableState> {
-                                      history.push(`/cases/edit/${id}`);
+                                  {
+                                      icon: () => (
+                                          <span aria-label="edit">
+                                              <EditIcon />
+                                          </span>
+                                      ),
+                                      tooltip: 'Edit this case',
+                                      // eslint-disable-next-line @typescript-eslint/no-explicit-any
+                                      onClick: (_: any, row: any): void => {
+                                          // Somehow the templating system doesn't think row has an id property but it has.
+                                          const id = (row as TableRow).id;
+                                          history.push(`/cases/edit/${id}`);
+                                      },
+                                      position: 'row',
                                   },
-                              },
-                          ]
-                        : []
-                    ).concat([
-                        {
-                            icon: () => (
-                                <span aria-label="details">
-                                    <VisibilityIcon />
-                                </span>
-                            ),
-                            tooltip: 'View this case details',
-                            onClick: (e, row): void => {
-                                // Somehow the templating system doesn't think row has an id property but it has.
-                                const id = (row as TableRow).id;
-                                history.push(`/cases/view/${id}`);
-                            },
-                        },
-                    ])}
+                                  // This action is for deleting selected rows.
+                                  // The action for deleting single rows is in the
+                                  // editable section.
+                                  {
+                                      icon: () => (
+                                          <span aria-label="delete all">
+                                              <DeleteIcon />
+                                          </span>
+                                      ),
+                                      tooltip: 'Delete selected rows',
+                                      // eslint-disable-next-line @typescript-eslint/no-explicit-any
+                                      onClick: (_: any, rows: any): void => {
+                                          rows.forEach((row: TableRow) =>
+                                              this.deleteCase(row),
+                                          );
+                                          this.tableRef.current.onQueryChange();
```

Comment:
Is it something about doing this within `actions` that necessitates this update?

For instance, with the `editable.onRowDelete` item, the table automatically refreshes based on the deletion.

Reply:
Yea the editable actions do that automatically but these actions do not.